src/gdash/scripts/_contains/_contains.gml

```diff
@@ -15,9 +15,10 @@ _contains("hello", "ello");
 // => true
 */
 
-var collection = argument[0];
-var target = argument[1]
-var fromIndex = 0;
+var 

```

Comment:
I'll change concat to make a new array then, no worries. I'll work on this again in ~23hrs and see how far you've come.